mws/mws.py

```diff
@@ -135,8 +135,14 @@ class DictWrapper(object):
 
     def __init__(self, xml, rootkey=None):
         if isinstance(xml, bytes):
-            xml = xml.decode()
-        self.original = xml
+            try:
+                xml = xml.decode(encoding="iso-8859-1")
```

Comment:
Always good to use the argument naming in the method call 👍 .

As we discussed on Slack for this issue, this is the encoding Amazon says they respond with, so it's good that we're closer to parity with this change now. Much appreciated.